python/cuxfilter/charts/core/non_aggregate/core_line.py

```diff
@@ -68,22 +75,13 @@ def __init__(
         self.pixel_shade_type = pixel_shade_type
         self.title = title
         self.timeout = timeout
+        self.x_axis_tick_formatter = x_axis_tick_formatter
+        self.y_axis_tick_formatter = y_axis_tick_formatter
         self.library_specific_params = library_specific_params
         self.width = width
         self.height = height
 
-    def initiate_chart(self, dashboard_cls):
-        """
-        Description:
-
-        -------------------------------------------
-        Input:
-        data: cudf DataFrame
-        -------------------------------------------
-
-        Ouput:
-
-        """
+    def compute_min_max(self, dashboard_cls):
         if type(dashboard_cls._cuxfilter_df.data) == dask_cudf.core.DataFrame:
```

Comment:
This pattern of conditioning on `== dask_cudf.core.DataFrame` to compute min/max etc happens at least three times. Suggest factoring things to a common set of functions that hides the switching on the underlying type.

Reply:
fixed, added a common [method](https://github.com/AjayThorve/cuxfilter/blob/new_dtypes/python/cuxfilter/charts/core/core_chart.py#L206-L209) to BaseChart class